src/IrisConvert/IrisConvert.php

```diff
@@ -101,6 +101,8 @@ public function createArrayOfSizesToBeConverted($metadata)
 
     public function convertArrayOfSizes()
     {
+        // Remove wp_generate_attachment_metadata action
+        remove_filter('wp_generate_attachment_metadata', [__CLASS__, 'convertImageOnUplodad'], 10);
```

Comment:
The `convertImageOnUpload` method is from the `Iris` class so think you'll need to use that as the reference rather than `__CLASS__` else that would reference the current class which doesn't contain the `convertImageOnUpload` method.


Likely something like: `remove_filter('wp_generate_attachment_metadata', [Iris::class, 'convertImageOnUplodad'], 10);` - but would need testing